main.nf

```diff
@@ -480,13 +490,16 @@ if (params.target_bed)          summary['Target BED']        = params.target_bed
 if (step)                       summary['Step']              = step
 if (params.tools)               summary['Tools']             = tools.join(', ')
 if (params.skip_qc)             summary['QC tools skip']     = skipQC.join(', ')
+if (params.trim_fastq)          summary['Fastq trim']        = "Fastq trim selected"
```

Comment:
May want to print other trimming presets too if `params.trim_fastq =  true` e.g.
https://github.com/nf-core/atacseq/blob/fa1e3f8993cd20e249b9df09d29c5498eff311d2/main.nf#L232-L240

Reply:
done.